packages/utils/lib/configuration/forms/index.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from 'zod';
+import { FORMS_CONFIG_OVERRIDE } from '../../../.ottehr_config';
+import { mergeAndFreezeConfigObjects } from '../helpers';
+
+const DEFAULT_FORMS_CONFIG: FormsConfig = {
+  forms: [
+    {
+      title: 'Texas Workers Compensation Form DWC073',
+      link: 'https://drive.google.com/file/d/1H2PLOAOlblpPJUkgmFvnw6XLJYRnW2GX/view?usp=drive_link',
+    },
+  ],
+};
+
+const mergedConfig = mergeAndFreezeConfigObjects(DEFAULT_FORMS_CONFIG, FORMS_CONFIG_OVERRIDE);
```

Comment:
I know we use merge config sometimes, but I guess it's almost always incorrect because in this case we receive an unpredictable state that contains elements from the default config and elements from the downstream config. 

I'm sure that if we want to modify something in the downstream config, we should simply define that config as desired and just use logic such as: use the downstream config if it exists, otherwise use the base config, without merging them.